src/MSBuild.Sdk.SqlProj.Aspire/DataTierApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,65 @@
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Lifecycle;
+using Microsoft.Build.Locator;
+using MSBuild.Sdk.SqlProj.Aspire;
+
+namespace Aspire.Hosting;
+
+public static class DataTierApplicationBuilderExtensions
+{
+    /// <summary>
+    /// Adds a data-tier application resource to the application based on a referenced project.
+    /// </summary>
+    /// <typeparam name="TProject">Type that represents the project that produces the data-tier application package.</typeparam>
```

Comment:
Maybe: Type that represents the project that produces the data-tier application package (.dacpac file).